api/metrics/escooter_metric.py

```diff
@@ -0,0 +1,127 @@
+from api.utils.database import rows_to_dicts
+
+
+class EscooterMetric:
+    """
+    Metrics for escooter travel metric
+    """
+
+    def __init__(self, con):
+        self.con = con
+        
+    def avg_distance_x_to_y(self):
+        """
+        Returns the average distance riders take throughout
+        the city
+        """
+        query = """
+        SELECT
+            start_community_area_number, end_community_area_number, avg_trip_distance
+        From 
+            Escooter
+        """.format()
+        cur = self.con.cursor()
+        cur.execute(query)
+        rows = rows_to_dicts(cur, cur.fetchall())
+        return rows
+
+    def avg_distance_based_on_start_can(self):
+        """
+        Returns the avg distance traveled based on starting community area numbers
+        """
+        query = """
+        SELECT
+            start_community_area_number, avg(avg_trip_distance)
```

Comment:
Use `AS` to rename the columns to area_number and value, so that they will match the column names the community view is expecting.